IdentityCore/src/claims/MSIDClaimsRequest.m

```diff
@@ -191,7 +191,7 @@ - (NSDictionary *)jsonDictionary
     
     for (NSNumber *target in self.claimsRequestsDict.allKeys)
     {
-        NSArray *requests = self.claimsRequestsDict[target];
+        NSSet *requests = self.claimsRequestsDict[target];
```

Comment:
Are you sure it's a set? @antrix1989 could you check?

Reply:
@xinyuxu1026 is right, it should be set here.